<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Convertor</class>
 <widget class="QWidget" name="Convertor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Base Convertor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/icons/images/icons/convertor.ico</normaloff>:/icons/images/icons/convertor.ico</iconset>
  </property>
  <widget class="QRadioButton" name="bit8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>8bit</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="bit16">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>16bit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Hex">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>HEX</string>
   </property>
   <widget class="QLineEdit" name="hexLine">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Dec">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>DEC</string>
   </property>
   <widget class="QLineEdit" name="signedLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>signed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>unsigned:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="unsignedLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Oct">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>OCT</string>
   </property>
   <widget class="QLineEdit" name="octLine">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Bin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>BIN</string>
   </property>
   <widget class="QLineEdit" name="binLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00000000</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
